code/__DEFINES/lazy_templates.dm

```diff
@@ -0,0 +1,13 @@
+#define LAZY_TEMPLATE_KEY_NUKIEBASE "LT_NUKIEBASE"
+#define LAZY_TEMPLATE_KEY_WIZARDDEN "LT_WIZARDDEN"
+#define LAZY_TEMPLATE_KEY_NINJA_HOLDING_FACILITY "LT_NINJAHOLDING"
+#define LAZY_TEMPLATE_KEY_ABDUCTOR_SHIPS "LT_ABDUCTORSHIPS"
+
+#define LAZY_TEMPLATE_KEY_LIST_ALL list( \
```

Comment:
Nit:

```suggestion
#define LAZY_TEMPLATE_KEY_LIST_ALL(...) list( \
```

...or make it a global list. Creating a list is not free or constant so it shouldn't be implied that it is.